data/transaction/status.go

```diff
@@ -26,13 +32,17 @@ func (tx TxStatus) String() string {
 
 // StatusComputer computes a transaction status
 type StatusComputer struct {
-	MiniblockType        block.Type
-	IsMiniblockFinalized bool
-	SourceShard          uint32
-	DestinationShard     uint32
-	Receiver             []byte
-	TransactionData      []byte
-	SelfShard            uint32
+	MiniblockType            block.Type
```

Comment:
why do we have all these fields exported?

Reply:
this structure will be refactored.